### DISCUSSION TITLE
how to properly implement a kubernetes linkerd cluster?

### DISCUSSION THREAD

**Discussion by Iceforestik** [linkerd/linkerd2#9846] [Help]
we have a kubernetes cluster, there is one service in it that has hostNetwork: true, I will say right away that it is necessary and not to use it. The problem is precisely in it, if you embed sidecar linkerd in all deployments, they will not be able to communicate with the service in which the hostNetwork parameter is true, that is, in the one where there is no sidecar linkerd. Is it possible to somehow teach other services to communicate with this service?

**Comment by mateiidavid** [comment]
Linkerd supports sending traffic to non-meshed workloads. You can disable sidecar injection on your deployment that runs with `hostNetwork: true`, your other meshed workloads will still be able to send traffic to it, you just won't have any mTLS or any metrics on the inbound/outbound side for the non-meshed pod.
